server/app/models/OrderModel.ts

```diff
@@ -0,0 +1,28 @@
+import { Document, Schema } from 'mongoose';
+import { materialTypes, MaterialTypes } from '../entities/Material';
+
+export interface IOrder extends Document {
+  materialType: materialTypes;
+  cost: number;
+  quantity: number;
+  deliveryDate: Date;
+  orderDate: Date;
+  manufacturerName: string;
+  status: string;
+  note: string;
+}
```

Comment:
@nasjuice can you add an attribute called 'location' from the location of the vendor?